src/features/styles/reset.scss

```diff
@@ -15,6 +15,7 @@ i {
 html {
     /* 1rem = 10px */
     font-size: 62.5%;
+    scroll-behavior: smooth;
```

Comment:
Do we need this part of code still @hasan-deriv ?

Reply:
removed

Reply:
We need it.